db/migrations/migrations_test.go

```diff
@@ -1,4 +1,4 @@
-// +build migrations
+//go:build migrations
```

Comment:
This is a change in directive style introduced in go 1.18:

https://stackoverflow.com/questions/68360688/whats-the-difference-between-gobuild-and-build-directives

This was an automated change from our pre-commit hook.